### DISCUSSION TITLE
Label callout annotation starting point

### DISCUSSION THREAD

**Discussion by MatHook** [chartjs/chartjs-plugin-annotation#687] [Q&A] [answered]
Hello, i'm kinda new in chart-js
I'm trying to have an annotation pointing to some segment of bar chart, but starting point in x axis are pointing on start/left edge of bar chart.
I've turned offset for x axis to false. And have stacked datasets in one bar.
![image](https://user-images.githubusercontent.com/31614310/154015905-5bedfee7-ffa8-4d41-b559-f09249d3826f.png)
Is it possible to specify starting point of annotation, like to right edge?
Thanks for reply in advance!

`"chart.js": "^3.6.0",
"chartjs-plugin-annotation": "^1.2.2"
`

**Comment by stockiNail** [comment, marked as answer]
@MatHook the callout element has got the `start` option where you can decide the position where the callout start from the side of the label.

You can set the number (pixel) or in percentage (as string). https://www.chartjs.org/chartjs-plugin-annotation/latest/guide/types/label.html#callout

In our cases, you could set the option `position` to `bottom` and set `start` to '0%'. 
I'm going to prepare a codepen to show that.

**Reply by stockiNail** [reply]
Here codepen with simple sample: https://codepen.io/stockinail/pen/rNYjqyx:
